### DISCUSSION TITLE
SSE(Server Sent Event)에 호환성 여부

### DISCUSSION THREAD

**Discussion by evan-hwang** [naver/ngrinder#889] [Q&A]
Spring WebFlux의 SSE 프로토콜을 사용하는 어플리케이션 테스트가 가능한지 문의들비니다.

**Comment by donggyu04** [comment]
[외부 라이브러리 import 하여 사용하기](https://github.com/naver/ngrinder/wiki/How-to-use-library) 를 통해 외부 라이브 러리를 스크립트 내에 import 하고 [다음](https://www.baeldung.com/java-ee-jax-rs-sse#consuming-sse-events) 과 같은 방식으로 스크립트 내에서 처리하면 되지 않을까 싶습니다.
